contribute/style-guides/themes.md

```diff
@@ -73,9 +79,9 @@ describe('MyComponent', () => {
 });
 ```
 
-### Using themes in Storybook
+### Using themes in [Storybook](https://storybook.js.org/)
 
-All stories are wrapped with `ThemeContext.Provider` using global decorator. To render `Themeable` component that's not wrapped by `withTheme` HOC you either create a new component in your story:
+All stories are wrapped with `ThemeContext.Provider` using a global decorator. To render a `Themeable` component that isn't wrapped by `withTheme` HOC, create a new component in your story.
```

Comment:
I think meaning gets lost with this edit since two alternatives are described below (new component vs `renderComponentWithTheme`):

Reply:
What do you think, @oddlittlebird ? https://github.com/grafana/grafana/pull/22381#discussion_r383377845

Reply:
Specifically, the "or" on line 95 is orphaned after the edit.

Reply:
If two components are described, then they need to be mentioned in the line 84 paragraph. The "or" that follows is so far away, it's not clear that it is continuing the previous thought. I would do something like this:

...HOC, you can create a new component in the story or do another thing.

**Thing 1**
Briefly explain Thing 1.
```
do code
```

**Thing 2**
Briefly explain Thing 1.
```
do code
```
---------
You can replace the bold above with headings.

Reply:
I like this solution better than the original. I'll make this change.

Reply:
Done: https://github.com/grafana/grafana/pull/22381/commits/75c6b09dc126c7ef240fc4ea23b45eb126e88ba8